docs/modbus_set_trace_handler.md

```diff
@@ -0,0 +1,44 @@
+# modbus_set_trace_handler
+
+## Name
+
+modbus_set_trace_handler - call method when log data is written
+
+## Synopsis
+
+```c
+typedef int (*modbus_stream_handler_t)(void *user, const char *format, va_list ap);
```

Comment:
this needs to document the return type, at least.  Again, "void *user" vs FILE * is ... muddy here.  I know you want void * user for when you have a custom handler, and file* magically working when you _just_ set the "user data" without registering a custom handler, but this is... _very_ unclear to people not well versed in these things.

Reply:
uses printf return type / values. I don't know how to document the void pointer stuff properly, this is a concept you have to understand in order to not make errors...